docs/dissemination/publications.md

```diff
@@ -0,0 +1,62 @@
+# Publications
+
+## Papers
+
+### 2022
+
+1. Pascal Reisert, Marc Rivinius, Toomas Krips and Ralf Kuesters: _Arithmetic
+Tuples for MPC_. In Cryptology ePrint Archive, Paper 2022/667. <br>
+:material-file-document-multiple-outline: [Technical Report](https://eprint.iacr.org/2022/667)
+
+2. Marc Rivinius, Pascal Reisert, Daniel Rausch and Ralf Küsters: _Publicly
+Accountable Robust Multi-Party Computation_. In 2022 IEEE Symposium on Security
+and Privacy (SP), San Francisco, CA, US, 2022, pp. 1520-1520. <br>
+:material-file-document-outline: [Conference Version](https://doi.ieeecomputersociety.org/10.1109/SP46214.2022.00091)
+&emsp; :material-file-document-multiple-outline: [Technical Report](https://eprint.iacr.org/2022/436)
+
+3. David Mestel, Johannes Müller, and Pascal Reisert: _How Efficient are Replay
+Attacks against Vote Privacy? A Formal Quantitative Analysis_. In 35th IEEE
+Computer Security Foundations Symposium, CSF 2022, Haifa, Israel, August 7-10,
+to appear. <br>
+:material-file-document-multiple-outline: [Technical Report](https://eprint.iacr.org/2022/743)
+
+4. Natasha Fernandes, Annabelle McIver, Catuscia Palamidessi, and Ming Ding.
+_Universal Optimality and Robust Utility Bounds for Metric Differential Privacy_
+. In 35th IEEE Computer Security Foundations Symposium, CSF 2022, Haifa, Israel,
+August 7-10, to appear. <br>
+:material-file-document-multiple-outline: [Technical Report](https://doi.org/10.48550/arXiv.2205.01258)
+
+## Talks
+
+### 2022
+
+1. Sven Trieflinger: _Carbyne Stack – Open-source cloud-native Secure Multiparty
+Computation_. Invited talk at Workshop on Theory and Practice of Multi-Party
+Computation, TPMPC 2022, Aarhus, Denmark, June 7-10. <br>
+:material-text-short: [Abstract](https://www.multipartycomputation.com/tpmpc-2022/invited-talks#h.bu4y0licb8g2)
+
+2. Sven Trieflinger: _Carbyne Stack - Cloud Native Computing on Encrypted Data_.
+Contributed talk at Emerging OS Forum, Open Source Summit North America 2022,
+OSS-NA 2022, Austin, Texas, USA, June 21-24. <br>
+:material-text-short: [Abstract](https://ossna2022.sched.com/event/11NhT) &emsp;
+:material-presentation: [Slides](https://static.sched.com/hosted_files/ossna2022/6b/20220623%20Carbyne%20Stack%20-%20Cloud%20Native%20Computing%20on%20Encrypted%20Data.pdf)
+
+## Software
+
+### 2021
+
+1. Robert Bosch GmbH: _Carbyne Stack - Cloud-native Secure Multiparty
```

Comment:
3 different versions (above) of "Cloud Native": "Cloud Native" "cloud-native" "Cloud-native".

Reply:
Fixed that in one place. For the others: these are the titles of the publications as they are. I guess there is no way to change them anymore and citing them means unfortunately also citing the inconsistent spelling.